delta-proto/src/main/java/io/cdap/delta/proto/DeltaConfig.java

```diff
@@ -191,6 +192,13 @@ public Stage getTarget() {
     if (sourceStage == null) {
       throw new IllegalArgumentException("No source found.");
     }
+
+    if (tableTransformations != null) {
```

Comment:
I think I missed in last PR review, but we should add `tableTransformations` to the equals and hashCode method as well.

Reply:
good catch :) Added equals and hashCode method for TableTransformations and ColumnTransformations too :)